mapping.md

```diff
@@ -39,6 +39,7 @@ documentation when migrating._
 | `last`           | `last`           | `last`              |
 | `map`            | `map`            | `map`               |
 | `mapKeys`        | `mapKeys`        | `-`                 |
+| `meanBy`         | `meanBy`         | `-`                 |
```

Comment:
in your PR description, you said that both `lodash` and `ramda` support `meanBy`, but you haven't added `ramda` to the mapping table 🤔